### ISSUE TITLE
temp file not deleted   if FILE_FORM_TEMP_STORAGE not set

### ISSUE THREAD

**Issue by sebhaase** [mbraak/django-file-form#349]
In line 50 of django_file_form/views/tus.py:
```
 50 B->     if hasattr(settings, 'FILE_FORM_TEMP_STORAGE'):
 51             os.remove(uploaded_file)
```
seems to be problematic.  In model.py (also) line 50 it defaults to `FileSystemStorage` which causes a temp file to be created.  I don't right a case, where the `uploaded_file` should not get deleted?

Thanks, Seb.

**Comment by mbraak** [comment]
Checked: the latest release (3.0.1) contains this error. But it  is already fixed in the development version.

**Comment by sebhaase** [comment]
Ref. PR #334

**Comment by mbraak** [comment]
This is fixed in the 3.1.0 release
